buildpipeline/DotNet-CLI-CentOS-x64.json

```diff
@@ -0,0 +1,172 @@
+{
```

Comment:
This is hopeful: Can we collapse these down to a single file for common file? having these files be disjoint will make it difficult to understand where our builds differ...